.github/workflows/build-everything.yml

```diff
@@ -45,3 +45,25 @@ jobs:
             openid-caep-specification-1_0.txt
             openid-risc-profile-specification-1_0.html
             openid-risc-profile-specification-1_0.txt
+  publish-to-pages:
+    if: github.ref == 'refs/heads/main'
```

Comment:
Could this be
```
if: github.ref == 'refs/heads/main' or github.ref == 'refs/heads/update-workflow'
```
so we could see this running on this branch?

Reply:
I've tested it using the update-workflow branch. I've now reverted it to the main branch